src/psyclone/psyir/nodes/operation.py

```diff
@@ -42,8 +42,11 @@
 from abc import ABCMeta
 from enum import Enum
 
-from psyclone.errors import GenerationError
+from psyclone.errors import GenerationError, InternalError
 from psyclone.psyir.nodes.datanode import DataNode
+from psyclone.psyir.symbols.datatypes import (
+    ArrayType, BOOLEAN_TYPE, DataTypeSymbol, DeferredType, ScalarType,
+    UnknownFortranType, UnknownType)
```

Comment:
pylint says unused InternalError and DatatypeSymbol

Reply:
Oops.